libraries/Microsoft.Bot.Builder.Dialogs/Prompts/OAuthPrompt.cs

```diff
@@ -191,8 +191,14 @@ public async Task<TokenResponse> GetUserTokenAsync(ITurnContext context)
             {
                 throw new InvalidOperationException("OAuthPrompt.GetUserToken(): not supported by the current adapter");
             }
-
-            if (magicCodeRegex.IsMatch(context.Activity.Text))

```

Comment:
magicCodeRegex should be named _magicCodeRegex because its a member variable

Reply:
fixed